src/app/services/org-team.service.ts

```diff
@@ -0,0 +1,52 @@
+import { Injectable } from '@angular/core';
```

Comment:
We don't need this service, since the job of this service is already being done in applicationSettingService.